controllers/managedmcg_controller_test.go

```diff
@@ -85,19 +63,31 @@ func newManagedMCGFake() mcgv1alpha1.ManagedMCG {
 	return managedMCGFake
 }
 
+func newSecretFake() corev1.Secret {
+	secretFake := corev1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "secretfake",
+			Namespace: "openshift-storage",
+		},
+	}
+	secretFake.Data = make(map[string][]byte, 1)
+	b := []byte("abc")
+	secretFake.Data["addonparam"] = b
+	return secretFake
+}
+
 func TestManagedMCGReconcilerReconcile(t *testing.T) {
 	r := &ManagedMCGReconciler{}
 	r.Log = ctrl.Log.WithName("controllers").WithName("ManagedMCGFake")
 	r.Scheme = newSchemeFake()
-	r.initializeReconciler(reconcile.Request{
-		NamespacedName: types.NamespacedName{
-			Name:      "managedmcgfake",
-			Namespace: "openshift-storage",
-		},
-	})
-	ODFCSVFake := newODFCSVFake()
+
 	managedMCGFake := newManagedMCGFake()
-	fakeClient := fake.NewClientBuilder().WithScheme(r.Scheme).WithObjects(&ODFCSVFake, &managedMCGFake).Build()
+	secretFake := newSecretFake()
+	r.AddonParamSecretName = "secretfake"
+	r.AddonConfigMapName = "configfake"
```

Comment:
Do we need `r.AddonConfigMapName = "configfake"`? Do the tests fail without this?

Reply:
No. Removed it.